specification/maps/data-plane/DataRegistry/stable/2022-12-01/examples/DataRegistries_Download.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-12-01",
+    "udid": "25084fb7-307a-4720-8f91-7952a0b91012"
+  },
+  "responses": {
+    "200": {
+      "body": "{file}"
```

Comment:
Maybe we could have a real example here.

Reply:
Fixed